tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py

```diff
@@ -99,10 +95,17 @@
         project_id=GCP_PROJECT_ID,
     )
 
+    # TEST BODY
     import_dataset_task >> create_model
+    # TEST TEARDOWN
     delete_model_task >> delete_datasets_task
 
     # Task dependencies created via `XComArgs`:
     #   create_dataset_task >> import_dataset_task
     #   create_dataset_task >> create_model
     #   create_dataset_task >> delete_datasets_task
+
```

Comment:
Seems like we need a watcher?

Reply:
Sure, we can have a watcher too for each of these tests